staging/src/k8s.io/client-go/transport/config.go

```diff
@@ -80,6 +84,11 @@ func (c *Config) HasCertAuth() bool {
 	return len(c.TLS.CertData) != 0 || len(c.TLS.CertFile) != 0
 }
 
+// IsDebugConcurrent returns whether we should be debugging concurrent state.
```

Comment:
Go accessors are usually of the form "X" as opposed to "GetX" or "IsX". Since DebugConcurrent is already public, do you really need this method?